src/controllers/index.js

```diff
@@ -1,6 +1,11 @@
 const Router = require('express').Router();
 const chat = require('./chat');
-
+const home = require('./home');
+const signUp = require ('./signUp');
```

Comment:
signin/up should not be exited in this pull request and also I don't see any file called signup.js or signin!!